testing/src/test/java/io/stargate/it/BaseOsgiIntegrationTest.java

```diff
@@ -73,15 +78,50 @@
     unFinal();
   }
 
-  static Starter stargateStarter;
-  public static String stargateHost = "127.0.0.11";
   public boolean enableAuth;
+
   protected final ClusterConnectionInfo backend;
 
+  public static List<Starter> stargateStarters = new ArrayList<>();
+  private static List<String> stargateHosts = new ArrayList<>();
+  public static final Integer numberOfStargateNodes = 3;
+
+  static {
+    for (int i = 1; i <= numberOfStargateNodes; i++) {
+      int portSuffix = 1 + i;
```

Comment:
`portSuffix = 10 + i`? I think we need to reserve a few lower address for `ccm`, which starts at `127.0.0.1` in our setup

Reply:
Please add a comment about that as well :)